docs/content/blog/persist-sensitive-data-safely.md

```diff
@@ -11,17 +11,17 @@ summary: |
     Keep sensitive data generated by Porter safe and sound with the new secret plugin protocol
 ---
 
-As we approaching closer to our v1 release, we have made another improvement in Porter to make sure any sensitive data generated or referenced by Porter is stored in a secure location.
-The newly updated secret plugin protocol enables Porter to persist sensitive data in a secret plugin store instead of Porter's storage database.
+As Porter approaches a v1.0.0 release, we have made an improvement in Porter to make sure any sensitive data generated or referenced by Porter is stored in a secure location.
+The newly updated secret plugin protocol enables Porter to securely store sensitive data in an external secret store instead of in Porter's database.
 
-Previously Porter only uses plugins for retrieving secrets from a secret store. When it comes to storing data generated by bundles, Porter uses storage plugins like Mongo as its backend database solution. If a sensitive data, such as a database connection string, were populated during a Porter execution, it would be stored in a Mongo database in plain text, which could lead to potential data breach.
-Now Porter requires users to configure a secret store to hold any data that has been flaged as sensitive in a bundle. 
+Previously Porter only uses plugins for retrieving secrets from a secret store. When it comes to storing data generated by bundles, Porter uses storage plugins like Mongo as its backend database solution. If a sensitive data, such as a database connection string, were generated by a bundle, it would be stored in a Mongo database in plain text.
```

Comment:
```suggestion
Previously Porter only uses plugins for retrieving secrets from a secret store. When it comes to storing data generated by bundles, Porter uses storage plugins like Mongo as its backend database solution. If sensitive data, such as a database connection string, were generated by a bundle, it would be stored in a Mongo database in plain text.
```